python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -38,73 +49,108 @@ class GaudiRun(IPrepareApp):
     The output from this command can be quite large and Ganga will save it to disk and store it at least once per (master) job
     If your build target is large I would advise that you consider placing your gangadir in your AFS workspace where there is more storage available
 
+
+    Usage:
+    ===============
+
     This application needs to be configured with the absolute directory of the project and the options you want to pass to gaudirun.py
 
     e.g.
 
     j=Job()
-    myApp = GaudiRun()
+    myApp = GaudiExec()
     myApp.directory = "$SOMEPATH/DaVinciDev_v40r2"
-    myApp.myOpts = "$SOMEPATH/DaVinciDev_v40r2/myDaVinciOpts.py"
+    myApp.options = "$SOMEPATH/DaVinciDev_v40r2/myDaVinciOpts.py"
     j.application = myApp
     j.submit()
 
     To setup a minimal application you can also run the helper function:
 
     prepare_cmake_app(myApp, myVer, myPath, myGetpack)
 
-    For GaudiPython style of running:
+
+    How it works:
+    ===============
 
     The actual command run on the WN is:
 
-        ./run gaudirun.py myOptsFile.py data.py
+        ./run gaudirun.py optionsFile.py data.py
```

Comment:
What exactly does this do? Does this impact what the user sees from `help(myApp)`?